### ISSUE TITLE
file:// link doesn't work

### ISSUE THREAD

**Issue by wedens** [emacs-lsp/lsp-ui#188]
When I click clickable `file://` link in the doc popup, I get "Point is not at a Markdown link or URL".

It was mentioned before in [another issue](https://github.com/emacs-lsp/lsp-ui/issues/176#issuecomment-426006150), but I still have the same problem with the latest lsp-ui (20181007.2219).

**Comment by sevanspowell** [comment]
Yes, I'm also still encountering this issue. I'm using the current develop branch of Spacemacs.

Is there any more information I could provide to help address this issue?

**Comment by acowley** [comment]
I dug into this a little bit, and the error is coming from [`markdown-mode`](https://github.com/jrblevin/markdown-mode/blob/master/markdown-mode.el#L7719). Perhaps our doc popup links are not valid markdown links, so its URI recognition and extraction are failing.

**Comment by yyoncho** [comment]
We have to either find/replace upfront or plug into markdown-mode.

**Comment by shajra** [comment]
This seems like a really nice feature to get working.  In the meantime, does anyone have a workaround that at least gets the link copy-able to paste in a brower?  Otherwise, my fallback is a tags file.

**Comment by ru13r** [comment]
I confirm the same issue.
file:// links cannot be opened from help-mode in the other buffer, despite the full address appears below the modeline on mouse hover.

**Comment by wraithm** [comment]
There's also this:

https://github.com/emacs-lsp/lsp-ui/issues/452
